qemu/tests/cfg/single_driver_install.cfg

```diff
@@ -7,9 +7,23 @@
     need_uninstall = no
     driver_install_timeout = 600
     tmp_folder = C:\tmp
+    nic_model_nic1 = rtl8139
     guest_alias = "Win7-32:w7\x86,Win7-64:w7\amd64,Win8-32:w8\x86,Win8-64:w8\amd64,Win8-32.1:w8.1\x86,Win8-64.1:w8.1\amd64,Win10-32:w10\x86,Win10-64:w10\amd64,Win2008-sp2-32:2k8\x86,Win2008-sp2-64:2k8\amd64,Win2008-r2-64:2k8R2\amd64,Win2012-64:2k12\amd64,Win2012-64r2:2k12R2\amd64,Win2016-64:2k16\amd64"
+    guest_alias_fl = "Win7-32:i386\Win7,Win7-64:amd64\Win7,Win8-32:i386\Win8,Win8-64:amd64\Win8,Win8-32.1:i386\Win8.1,Win8-64.1:amd64\Win8.1,Win10-32:i386\Win10,Win10-64:amd64\Win10,Win2008-sp2-32:i386\Win2008,Win2008-sp2-64:amd64\Win2008,Win2008-r2-64:amd64\Win2008R2,Win2012-64:amd64\Win2012,Win2012-64r2:amd64\Win2012R2,Win2016-64:amd64\Win2016"
     driver_install_cmd = "python WIN_UTILS:\win_driver_install.py %s %s %s %s %s"
     show_file_ext_cmd = "reg add HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced /v HideFileExt /t REG_DWORD /d 0 /f"
+    vol_virtio_key = "VolumeName like '%virtio-win%'"
+    variants:
+        - @default:
+        - with_floopy:
+            run_with_floopy = yes
+            vol_virtio_key = "DeviceId like 'A:'"
+            floppy_readonly = yes
+            floppies = "floopy"
```

Comment:
It seems these two params are not used.

Reply:
Hi hachen, the param "floppies =  " means will add a floppy disk to guest, detail image depending on param "floppy_name = "; and the "floopy_readonly = yes" will determine the added floopy disk including the attribute "readonly=on", the default param "floopy_readonly = no".

Thanks for your review this patch~ 
This patch will not be merged, because it based on the case single_driver_install, but the single_driver_install is modifying by xuhan due do some issues, he will add this patch's function to the new single_diver_install patch~